MTGToolbox.Repository/UnitOfWork.cs

```diff
@@ -1,16 +1,13 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-using Microsoft.EntityFrameworkCore;
+﻿using Microsoft.EntityFrameworkCore;
+using MTGToolbox.Abstract;
 
 namespace MTGToolbox.Repository
 {
-    class UnitOfWork : IDisposable
+    internal class UnitOfWork
```

Comment:
Sorry this should be public or something. I have my VS set up to do some aggressive clean up.